alfasim_sdk_api/detail/api_pointers.h

```diff
@@ -59,6 +59,37 @@ get_plugin_input_data_string_func get_plugin_input_data_string;
 typedef int (*get_plugin_input_data_string_size_func)(void*, int*, const char*, const char*);
 get_plugin_input_data_string_size_func get_plugin_input_data_string_size;
 
+/**
+*   get_plugin_input_data_table_quantity
+*
+*   Get the values from a column of an input table. Column_id is the string defined in the plugin's
+*   configuration file. If the var_name or the column_id are invalid, UNDEFINED_DATA is returned.
+*
+*   This function is only available for the following hooks:
+*   - HOOK_INITIALIZE
+*
+*   Example usage:
+*   int size = -1;
+*   double* values = NULL;
+*   int errcode = get_plugin_input_data_table_quantity(
+*       ctx,
+*       &values,
+*       &size,
+*       "temperature",
+*       plugin_name,
+*       "some_table"
+*    );
```

Comment:
Would be nice to show an example of how to actually use `values` (together with `size` of course)`

Reply:
Sure

Reply:
Done.